lib/rack/attack.rb

```diff
@@ -53,6 +53,19 @@ def clear!
         @configuration.clear_configuration
       end
 
+      def reset!
+        store = cache.store
+
+        if store.respond_to?(:delete_matched)
+          store.delete_matched("#{cache.prefix}*")
+        else
```

Comment:
Sounds like everything in this method implementation is interacting with `cache`-only stuff (`store` and `prefix`).
Should we move `reset!` inside `Cache` and make the implementation here just `cache.reset!`?